dd-sdk-android/src/main/kotlin/com/datadog/android/core/configuration/Configuration.kt

```diff
@@ -341,7 +343,14 @@ internal constructor(
          * @see [Feature.Tracing]
          * @see [Feature.RUM]
          */
-        fun addPlugin(plugin: DatadogPlugin, feature: PluginFeature): Builder {
+        @Deprecated(
+            "Datadog Plugins won't work in SDK v2, you'll need to write" +
+                " your own Feature"
```

Comment:
```suggestion
            "Datadog Plugins will be removed in SDK v2.0.0." +
                "You will then need to write your own Feature (check our own code for guidance)."
```

Reply:
The problem is that there is no code to check with the guidance, it will be available (with all the public API changes) in version 2.0.0, so that it means we cannot remove things related to the `DatadogPlugin` in 2.0.0 and should do it later at some point. WDYT? In theory it is not a problem to keep `DatadogPlugin` a bit longer. Or we don't add the statement about checking the code.